agent_api/src/main/java/dev/aikido/agent_api/background/HeartbeatTask.java

```diff
@@ -9,6 +9,8 @@
 import dev.aikido.agent_api.storage.routes.RouteEntry;
 import dev.aikido.agent_api.context.User;
 import dev.aikido.agent_api.storage.routes.RoutesStore;
+import dev.aikido.agent_api.storage.statistics.Statistics;
+import dev.aikido.agent_api.storage.statistics.StatisticsStore;
```

Comment:
unused?

Reply:
no they are used, probably a merge conflict, also this PR should be reviewed after the mutex stuff